inference-engine/ie_bridges/python/sample/style_transfer_sample/style_transfer_sample.py

```diff
@@ -135,7 +136,10 @@ def main():
             output_image = cv2.resize(output_image, (w, h))
 
         cv2.imwrite(f'out_{i}.bmp', output_image)
-        log.info(f'Image out_{i}.bmp created!')
+        if os.path.exists(f'out_{i}.bmp'):
+            log.info(f'Image out_{i}.bmp created!')
+        else:
+            log.error(f'Image out_{i}.bmp was not created. Check your permissions.')
```

Comment:
Why image cannot be created?

Reply:
If no write permission in current directory, out.bmp is not created but [ INFO ] output claims it was created.